statsmodels/tsa/stattools.py

```diff
@@ -1330,10 +1330,10 @@ def grangercausalitytests(x, maxlag, addconst=True, verbose=True):
     >>> import numpy as np
     >>> data = sm.datasets.macrodata.load_pandas()
     >>> data = data.data[["realgdp", "realcons"]].pct_change().dropna()
-
+    >>>
```

Comment:
This isn't needed.

Reply:
I'm not that familiar with Git, is that possible to change the PR?

Reply:
never mind. I guess I fixed it as requested.